src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -31,21 +30,26 @@
                 token)]
     result))
 
+(re-frame/reg-sub
+ :wallet/accounts
+ :<- [:wallet]
+ (fn [{:keys [accounts]}]
+   (->> (vals accounts)
+        (sort-by :position))))
```

Comment:
I'm not a big fan of the threading macros that only perform one step, I see this as if it'd be the same as:
`(->> {:a 1} :a)`, in this case it's better to just do `(:a {:a 1})` IMO.

What about this?:
```clojure
(re-frame/reg-sub
 :wallet/accounts
 :<- [:wallet]
 (fn [{:keys [accounts]}]
   (->> accounts vals (sort-by :position))))

;; Also can be written as:
(re-frame/reg-sub
 :wallet/accounts
 :<- [:wallet]
 :-> #(->> % vals (sort-by :position)))
```

just a suggestion, nothing important :)

Reply:
I have no strong opinion on this. Since we use only one key from app-db (or perform one step), I'm fine with the suggestion. 👍